infrastructure/idam-preview.tfvars

```diff
@@ -6,4 +6,4 @@ asp_rg_override = "idam-idam-preview"
 
 capacity = 1
 
-strategic_policies_privateIpsFilterPattern = ""
\ No newline at end of file
+vnet_private_ip_pattern = "10\\.97\\.\\d+\\.\\d+"
```

Comment:
Conversely, I think retaining the same name make it much easier to identify mistakes and issues in the marshalling of variables through the provisioning tasks.

@kremi

Reply:
Yeah, that makes sense and is a different way to look at it. 

My thinking is it’s helpful to have the var name be meaningful in the current context. Having it as vnet private ip makes it easier for me to read and reason about in the context of infrastructure provisioning.